src/common/models/Projection.ts

```diff
@@ -39,7 +39,9 @@ const DISABLED_ICU = new DisabledFips([
   '53', // https://trello.com/c/1IkmUuhw/
 ]);
 
-const DISABLED_VACCINATIONS = new DisabledFips([]);
+const DISABLED_VACCINATIONS = new DisabledFips([
+  /39.../, // Disabling Ohio counties - vaccinations showing 0%
```

Comment:
Just noting that we know this is caused by a mistake in the scrapers treating a daily metric as a cumulative.

Reply:
yeah i'll replace this with the trello ticket